neurom/fst/__init__.py

```diff
@@ -85,7 +85,8 @@
     'segment_taper_rates': _nrt.segment_taper_rates,
     'segment_radial_distances': _nrt.segment_radial_distances,
     'segment_meander_angles': _nrt.segment_meander_angles,
-    'principal_direction_extents': _nrt.principal_direction_extents
+    'principal_direction_extents': _nrt.principal_direction_extents,
+    'total_surface_area_per_neurite': _nrt.total_surface_area_per_neurite
```

Comment:
not sure why principal_direction_extents didn't have a trailing comma, but could you add one to the line you added?